src/chttpd/src/chttpd_db.erl

```diff
@@ -569,15 +569,23 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_bulk_docs">>], user_ctx=Ctx}=Req,
             send_json(Req, 417, ErrorsJson)
         end;
     false ->
-        case fabric:update_docs(Db, Docs, [replicated_changes|Options]) of
-        {ok, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 201, ErrorsJson);
-        {accepted, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 202, ErrorsJson)
+        Docs2 = [Doc || Doc <- Docs, element(3, Doc) =/= {0, []}],
+        case length(Docs2) of
+        0 ->
+            send_json(Req, 400, {[{missing_revs,
+                ?l2b("If `new_edits` is false, a well-formed "
+                "`_rev` must be included in the document.")}]});
+        _ ->
```

Comment:
I had noticed in the previous incarnation of the PR https://github.com/apache/couchdb/pull/3812 that we should really double check if _rev is actually needed. `replicated` changes should primarily rely on `_revisions` so that would make that error incorrect. (But do double-check the semantics and interplay between those).